core/app/models/spree/permission_set.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Spree
+  class PermissionSet < Spree::Base
+    has_many :role_permissions
+    has_many :roles, through: :role_permissions
+    validates :name, :set, presence: true
+    scope :display_permissions, -> { where('name LIKE ?', '%Display') }
```

Comment:
If we would introduce a `permission_type` (still haven't came up with a better name) column (as proposed above) we could scope this by this column and would not rely on class name match here.

Reply:
Also done, we have introduced `category` and `privilege`, another good call thanks for the push!

Reply:
Should we update the scopes and make use of `privilege` instead of the `name` here?